components/app/SponsorCarousel.client.vue

```diff
@@ -15,6 +18,11 @@ import 'swiper/css'
     </h2>
     <Swiper
       :space-between="50"
+      :autoplay="{
+        delay: 5000,
```

Comment:
nit pick, I think I maybe like 3 seconds more? Cause 5 feels like too long and then I scroll by, maaayyybe even 2.5 but that seems a little too much

Reply:
Sorry just to clarify do you mean increasing it by 3s (so 8000ms) or reducing to 3s (3000).

Reply:
Nevermind I re-read and realized what you meant... lol